package.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
The package.json file is essential for managing dependencies and scripts in a Node.js project 🚀. It seems that the entire file was removed. Please restore the package.json file to the repository.